cpp/tests/CMakeLists.txt

```diff
@@ -182,6 +182,7 @@ ConfigureTest(DATETIME_OPS_TEST datetime/datetime_ops_test.cpp)
 # * hashing tests ---------------------------------------------------------------------------------
 ConfigureTest(
   HASHING_TEST
+  hashing/bloom_filter_test.cu
```

Comment:
Maybe this goes better with Parquet tests? Or a separate test file?

Reply:
Yes, let me move this to parquet for now. Eventually we want to migrate it to cuco.